lib/cypress-rails/init.rb

```diff
@@ -19,8 +19,8 @@ class Init
       })
     JS
 
-    def call(dir = Dir.pwd)
-      config_path = File.join(dir, "cypress.config.js")
+    def call(cypress_dir = Config.new.cypress_dir)
```

Comment:
Needed to also include this here, to make sure we're looking to write the `"cypress.config.js"` to the cypress dir